src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/LetterControllerITest.java

```diff
@@ -699,16 +711,16 @@ void jurorNameSearchIncludePrinted() {
                     .isEqualTo(OK);
                 assertThat(response.getBody()).isNotNull();
                 List<?> responseBody = response.getBody().getData();
-                assertThat(responseBody.size()).isEqualTo(2);
+                assertThat(responseBody.size()).isEqualTo(1);
```

Comment:
Why would these numbers be different?

Reply:
There is actually only one record that is printed in the setup data - it was erroneously showing 2 before